### PR TITLE
Improvement of jaggery apps

### PR DESCRIPTION
**Pull request by IshikaGodage**
Handled errors in jaggery apps by redirecting to an error page.

### CODE REVIEW

File: modules/jaggery-apps/portal/error_pages/error.jag [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isharailanga) [inline modules/jaggery-apps/portal/error_pages/error.jag]: 2019

Comment (IshikaGodage) [reply modules/jaggery-apps/portal/error_pages/error.jag]: Updated the year

File: modules/jaggery-apps/portal/gadgets/backup_otp/index.jag [hunk 2]
```diff
@@ -15,8 +15,8 @@ function getUser(){
 
 		print( fieldValues );
 	}catch(e){
-		print(e);
-		log.error(e);
+        log.error(e);
```
Comment (vihanga-liyanage) [inline modules/jaggery-apps/portal/gadgets/backup_otp/index.jag]: Fix the formatting pls.

File: modules/jaggery-apps/portal/error_pages/error.jag [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<html>
+    <head>
+        <title>Error</title>
+    </head>
+    <body>
+        <h1>Error Occurred</h1>
```
Comment (vihanga-liyanage) [inline modules/jaggery-apps/portal/error_pages/error.jag]: Did we discuss on proper design of this page? What should be displayed, etc?

**Pull request merged**